### PR TITLE
Add Pinata

### PR DESCRIPTION
**Pull request by soodaayush**
There doesn't seem to be documentation about the 2FA setting. What is an alternative for this?

### CODE REVIEW

**Comment by hkamran80** [comment]
If a documentation URL is not publicly available, upload a screenshot of the setup process or usage.

File: img/p/pinata.cloud.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...1 1.592-5.339 2.904-8.991 3.934a42.362 42.362 0 0 1-10.818 1.405c-8.335 0-14.751-1.78-19.247-5.339-4.495-3.559-6.79-9.272-6.883-17.139V185.2h-12.082v-16.999h12.082v-15.735l23.882-6.603v22.338h20.933V185.2H680.34v28.238c0 6.275 3.091 9.413 9.273 9.413 3.933 0 7.352-1.031 10.255-3.091l2.529 14.751ZM781.258 224.677c1.686 0 3.044-.187 4.074-.562l-1.405 13.627c-2.81 1.312-6.228 1.967-10.256 1.967-4.589 0-8.429-.843-11.52-2.529-3.09-1.779-5.198-4.402-6.322-7.867-1.873 3.372-4.917 5.994-9.131 7.867-4.215 1.78-8.804 2.67-13.768 2.67-7.305 0-13.065-1.639-17.28-4.917-4.215-3.372-6.322-8.196-6.322-14.471 0-10.396 6.322-16.905 18.966-19.527l24.304-5.058v-1.967c0-3.278-.936-5.853-2.809-7.727-1.78-1.966-4.215-2.95-7.306-2.95-6.837 0-11.005 3.512-12.503 10.537l-20.23-4.777c1.311-6.931 4.823-12.363 10.536-16.296 5.807-3.934 13.674-5.901 23.602-5.901 10.302 0 18.263 2.295 23.883 6.884 5.713 4.589 8.57 11.192 8.57 19.809v25.709c0 3.653 1.639 5.479 4.917 5.479Zm-40.742-.281c3.466 0 6.322-.983 8.57-2.95 2.342-2.061 3.512-5.058 3.512-8.991v-3.934l-11.379 2.388c-2.904.656-5.011 1.546-6.322 2.669-1.311 1.031-1.967 2.576-1.967 4.637 0 2.06.702 3.605 2.107 4.636 1.405 1.03 3.232 1.545 5.479 1.545Z"/></svg>
```
Comment (hkamran80) [inline img/p/pinata.cloud.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 256.1 379.02"><path fill="#8000db" d="M50.52 194.76c-1.38-7.85-3.04-15.44-6.91-21.57-5.25-8.27-14.51-13.92-32.76-14.06l-.28 10.34c6.5-.2 12.72.41 18.04 3.79 4.5 2.83 8.3 7.51 11.2 15.3-15.9-5.38-24.88-3.79-31.3.35-3.33 2.13-5.95 4.96-8.51 7.85l7.19 6.62c.41-.62.9-1.17 1.31-1.66 7.74-9.16 16.73-8.89 26.26-3.85-20.6 9.85-23.9 27.22-22.8 39.7.96.13 10.09 0 10.09 0-.9-12.2 7.8-28.54 25.91-33.43l2.7-5.52z"/><path fill="#008298" d="M185.98 87.45c30.48-42.18 45.55-63.07 33.45-73.2-7.95-6.68-18.73-1.93-36.14 11.03-.63 13.85-4.01 33.57-8.43 59.69z"/><path fill="#580098" d="M205.26 340.26h-11.2c-.27 15.92.15 32.33-16.92 37.57l1.8.2c1.79.21 3.59.07 5.31-.34 21.22-5.38 20.74-21.64 21.01-37.43z"/><path fill="#98005e" d="M217.36 340.26l-11.82-27.84h10.99l-9.26-26.26h-15.28l7.12 26.26h-8.43l9.12 27.84z"/><path fill="#8000db" d="M177.13 377.83c17.08-5.24 16.66-21.58 16.94-37.57H153.5c-.14 23.71 8.43 35.3 23.63 37.57z"/><path fill="#ff009e" d="M190.68 312.42h8.43L192 286.16l-41.19-.2-9.6 27.28 10.43.83-10.43 26.2h58.6z"/><path fill="#98005e" d="M105.19 313.38l-10.92-27.64-15.34-.06 7.53 27.08h-8.64l9.33 28.12 19.35.21-13.48-27.84z"/><path fill="#ff009e" d="M77.82 312.76h8.64l-7.53-27.08-41.82-.21-10.09 28.12 10.65.83-10.65 26.32 60.13.14-9.33-28.12z"/><path fill="#00bedd" d="M173.26.4c-16.38-4-25.98 22.2-49.76 73.2l51.42 11.37c4.43-26.12 7.81-45.84 8.43-59.69.63-14.82-2-22.88-10.09-24.88z"/><path fill="#f90" d="M204.71 259.2l13.55.9-14.45-28.74h11.89l-10.71-26.12h-17.14l9.67 26.12h-10.7l12.98 28.74-12.23-.9 13 26.96 18.52.07z"/><path fill="#008298" d="M218.26 178.22l-9.95-27.84h-19.29l8.23 27.84h-11.89l10.44 27.02h20.73l-12.72-27.02z"/><path fill="#ffe001" d="M123.5 73.6c-.06 0-.06.13 0 0l-22.87 45.28 12.65.28-15.2 31.22h109.95l-14.24-6c-2.14-.9-3.45-2.62-2.9-3.79.07-.14.14-.2.2-.28.7-.96 2.63-1.1 4.57-.27l.62.27 9.95 4.2 8.64 3.66V93.86z"/><path fill="#fff" d="M178.86 125.29c6.91 0 12.58-5.58 12.58-12.55 0-6.89-5.6-12.54-12.58-12.54a12.56 12.56 0 00-12.58 12.54c0 6.97 5.67 12.55 12.58 12.55zm0-18.82a6.3 6.3 0 016.3 6.28 6.3 6.3 0 01-6.3 6.27 6.3 6.3 0 01-6.29-6.27 6.3 6.3 0 016.3-6.28z"/><path fill="#000" d="M179.26 119a6.28 6.28 0 10-.76-12.53 6.28 6.28 0 00.76 12.54zm35.61 31.38v-2.2l-18.6-7.94-.61-.27c-1.94-.83-3.87-.69-4.56.28-.07.06-.14.2-.21.27-.49 1.17.83 2.9 2.9 3.8l14.24 5.99h.27z"/><path fill="#00bedd" d="M185.36 178.22h11.82l-8.16-27.84h-77.54l-11.75 27.84h11.82l-11.13 26.95h.14v.07h95.24z"/><path fill="#ffe001" d="M100.35 205.17l-49.62-6.68-15.2 30.8h10.02c-.97 1.31-2 2.7-2.98 4.14l13.96 13.03 18.11-17.1 17.62 17.51 18.11-17.1 18.11 17.1 17.62-17.5.56.54 17.48 16.48 18.11-17.1 6.91 6.83-2.14-4.83h10.71l-9.67-26.19h-87.57"/><path fill="#ffe001" d="M187.57 259.2l12.23.84-2.48-5.45-15.28-15.16L164 257.42l-18.1-18-18.04 18.07-17.7-17.65-18.1 17.1-17.49-17.37-18.04 17.98-18.03-17.92a640.38 640.38 0 00-10.65 18.82l13.76.83-12.24 26.19 171.2.76z"/><path fill="#000" d="M189.02 236.19l-6.9-6.83-18.12 17.1-18.1-17.1-17.63 17.51-18.1-17.1-18.11 17.1-17.63-17.5-18.1 17.09-13.96-13.03c-1.32 1.93-2.7 4-4.01 6.2l18.04 17.92 18.03-17.99 17.49 17.37 18.1-17.09 17.7 17.64 18.1-17.98 18.12 17.98 18.03-17.98 15.28 15.16z"/><path fill="#580098" d="M81.55 340.81c-.27 16.06.21 32.54-16.93 37.84 1.04.14 2.07.28 3.18.35a9.36 9.36 0 002.76-.2l2.35-.63c1.1-.34 2.22-.69 3.18-1.03a54.42 54.42 0 001.94-.83c.41-.2.83-.41 1.17-.62.35-.2.76-.41 1.1-.62a31.8 31.8 0 001.6-1.03c.34-.21.62-.49.97-.7a21.8 21.8 0 001.93-1.65 45.23 45.23 0 001.8-1.93c.2-.27.41-.48.62-.76a30.74 30.74 0 001.87-3.1c.2-.34.34-.76.55-1.1.07-.2.2-.41.27-.62.14-.28.21-.62.35-.9a.98.98 0 00.14-.48c.2-.62.41-1.24.55-1.86 0-.14.07-.28.07-.34.14-.42.2-.83.28-1.24 0-.07.07-.21.07-.28.07-.48.2-.97.27-1.45.07-.27.07-.55.14-.82.07-.28.07-.63.14-.9a40 40 0 00.41-4.14c.14-1.72.21-3.44.21-5.17.07-1.86.07-3.72.14-5.58z"/><path fill="#8000db" d="M40.98 340.75c-.07 4.61.21 8.82.83 12.54.21 1.45.49 2.82.83 4.13.21.9.49 1.73.7 2.55 3.52 11.17 10.78 17.03 21.21 18.61 17.14-5.3 16.66-21.78 16.93-37.83z"/><path fill="#98005e" d="M255.78 122.8c-4.14 10.35-17.76 18.76-40.98 20.82v6.69c1.6-.07 3.04-.2 4.56-.28a59.16 59.16 0 005.19-.62c1.52-.2 2.97-.48 4.35-.76 6.29-1.3 11.47-3.24 15.62-5.65.9-.55 1.8-1.1 2.56-1.65.41-.28.76-.55 1.17-.9a30.03 30.03 0 003.04-2.82c.63-.62 1.1-1.31 1.6-2a19.97 19.97 0 002.62-5.31c.69-2.34.76-4.9.28-7.51z"/><path fill="#ff009e" d="M219.85 95.1c-1.6-.41-3.25-.9-4.98-1.24v49.83c23.22-2.07 36.84-10.54 40.98-20.81-2.07-10.62-14.1-21.65-36-27.78z"/></svg>
```

File: entries/p/pinata.cloud.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Pinata": {
+    "domain": "pinata.cloud",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://app.pinata.cloud/profile",
```
Comment (hkamran80) [inline entries/p/pinata.cloud.json]:
```suggestion
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Here is the setup image:

setup

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to server and browser caches. In the future, make sure to blur out any sensitive information in any screenshots.

**Pull request merged**